GitUI/CommandsDialogs/SettingsDialog/Pages/FormBrowseRepoSettingsPage.ResourceImagesTypeConverter.cs

```diff
@@ -0,0 +1,31 @@
+﻿#nullable enable
+
+using System.ComponentModel;
+using System.Linq;
+using GitUI.Shells;
+
+namespace GitUI.CommandsDialogs.SettingsDialog.Pages
+{
+    public partial class FormBrowseRepoSettingsPage
+    {
+        private sealed class ResourceImagesTypeConverter : StringConverter
```

Comment:
`ResourceImagesTypeConverter` and `ResourceImagesTypeEditor` need to be moved out of `FormBrowse*` namespace, they are general purpose infra, that we should be able to leverage for other things, like the Scripts.